src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -173,6 +190,49 @@ internal LbfgsTrainerBase(ArgumentsBase args, IHostEnvironment env, string name,
                     ch.Done();
                 }
             }
+            _srcPredictor = default;
+        }
+
+        internal LbfgsTrainerBase(IHostEnvironment env, TArgs args)
```

Comment:
>LbfgsTrainerBase [](start = 17, length = 16)

I think you can this constructor from the other one to get ride to code redundancy.